### DISCUSSION TITLE
[WP4018 Discussion]:

### DISCUSSION THREAD

**Discussion by KestUrba** [wikipathways/wikipathways-help#54] [General]
### Discussion

This is useful information in WP4018, stressing enhanced glycolysis and lipid synthesis, but there are much more  signaling and metabolic pathways in Clear cell renal carcinomas. The true name of this pathway must  be "Clear cell renal carcinoma RTK-PI3K...mTOR1 pathway".
According to ref. Zhang J, Zhang Q. **VHL and Hypoxia Signaling: Beyond HIF in Cancer.** Biomedicines. 2018 Mar 19;6(1):35. doi: 10.3390/biomedicines6010035. PMID: 29562667; PMCID: PMC5874692.  there exist some important VHL regulated signaling axes. Unfortunately they were not described! 

### Should this pathway be deleted?

- [ ] Yes, this pathway should be deleted

**Comment by khanspers** [comment]
Thanks @KestUrba. I've opened a curation task for this proposed work here: https://github.com/wikipathways/pathway-curation-tasks/issues/32
